deepinv/loss/regularisers.py

```diff
@@ -39,14 +40,36 @@ class JacobianSpectralNorm(Loss):
         tensor([49.0202])
     """
 
-    def __init__(self, max_iter=10, tol=1e-3, eval_mode=False, verbose=False):
+    def __init__(self, max_iter=10, tol=1e-3, eval_mode=False, verbose=False, reduction='max'):
         super(JacobianSpectralNorm, self).__init__()
         self.name = "jsn"
         self.max_iter = max_iter
         self.tol = tol
         self.eval = eval_mode
         self.verbose = verbose
 
+        self.reduction = lambda x: x
```

Comment:
I think add an 
```python
else:
    raise ValueError('reduction should be "mean", "sum", "max", "none" or None.')
```
Also sorry to be annoying, but please can you also add a "none" option? Just to be consistent with https://deepinv.github.io/deepinv/stubs/deepinv.loss.metric.Metric.html#deepinv.loss.metric.Metric